### DISCUSSION TITLE
Global Styles per template

### DISCUSSION THREAD

**Discussion by arashsafaridev** [WordPress/gutenberg#52086] [Ideas]
Global styles currently apply across the entire site. It would be useful if global styles could be saved on a per-template basis. 

When saving global styles, the “Custom Styles” options are provided. An additional option could be “Template Styles” that allows the user to set a `wp_template` post ID as the parent for `wp_global_styles`.

This would enable users to define different styles for each template.

**Comment by carlomanf** [comment]
It looks like there are already issues for this at #44997 and #51384.

**Reply by arashsafaridev** [reply]
Thank you @carlomanf 
This is great.
